src/free-trial/tax/woocommerce-tax/plugins.tsx

```diff
@@ -57,9 +57,9 @@ export const Plugins: React.FC< SetupStepProps > = ( {
 		nextStep();
 	}, [ isResolving ] );
 
-	const agreementText = pluginsToActivate.includes( 'woocommerce-services' )
+	const agreementText = pluginsToActivate.includes( 'woocommerce-tax' )
```

Comment:
Currently the `Plugins` component isn't used since we're expecting bridge to always have `woocommerce-services` installed ([Removal commit](https://github.com/Automattic/wc-calypso-bridge/commit/6a431aec695799ea196f31b701f232327fbe1fe0#diff-b34996ead2d86a45c7ebced6cd02928d1cca579483652c45f4a48e382f06858bL104-L114)). Let me know if installing plugin in the task is required!